src/js.cookie.js

```diff
@@ -136,17 +139,15 @@
 			return key ? jar[key] : jar;
 		}
 
-		api.set = api;
+		api.set = set;
 		api.get = function (key) {
-			return api.call(api, key);
+			return get(key, false);
```

Comment:
[Boolean Trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap)?

Reply:
Maybe because it's internal function we can use a constant to name the argument. Should have no big deal in byte size because minifiers should be able to strip the constant out.

Reply:
Unfortunately Uglify - at least the version used here - didn't inline the variable so I added the comment instead.